src/XMakeBuildEngine/Definition/ToolsetReader.cs

```diff
@@ -240,8 +243,9 @@ ToolsetDefinitionLocations locations
                         }
                     }
                 }
+#if FEATURE_WIN32_REGISTRY
```

Comment:
Because of the comment above, this should not be required either, just drop the extra brace.